addons/event/views/event_event_views.xml

```diff
@@ -350,19 +350,11 @@
 
     <!-- EVENT.EVENT HEADER: REGISTRATION DESK MENU -->
 
-    <record id="event_action_install_kiosk_pwa" model="ir.actions.client">
-        <field name="name">Registration Desk</field>
-        <field name="target">new</field>
-        <field name="res_model">event.event</field>
-        <field name="tag">install_kiosk_pwa</field>
-        <field name="context">{'app_id': 'event', 'footer': false}</field>
-    </record>
```

Comment:
Hello @nihp-odoo 
Not sure about the upgrade :thinking: will let the reviewers decide about this :slightly_smiling_face: